src/stare/cli.py

```
@@ -0,0 +1,942 @@
+    """Show current authentication status (quick check)."""
+    tm = _make_token_manager()
+    if tm.is_authenticated():
+        console.print("[green]Authenticated[/green]")
+    else:
+        console.print(
+            "Not authenticated. Run [bold]stare auth login[/bold] to authenticate."
+        )
+
+
+@auth_app.command("info")
+def auth_info(
```

Ruff (0.15.10)

[warning] Too many branches (21 > 12)

(PLR0912)